files/downloads/index.html

```diff
@@ -5,127 +5,501 @@
-  <a href='/files/downloads/1.7.21/index.html'>1.7.21</a><br/>
-  <a href='/files/downloads/1.7.22/index.html'>1.7.22</a><br/>
-  <a href='/files/downloads/1.7.23/index.html'>1.7.23</a><br/>
-  <a href='/files/downloads/1.7.24/index.html'>1.7.24</a><br/>
-  <a href='/files/downloads/1.7.25/index.html'>1.7.25</a><br/>
-  <a href='/files/downloads/1.7.26/index.html'>1.7.26</a><br/>
-  <a href='/files/downloads/1.7.27/index.html'>1.7.27</a><br/>
-  <a href='/files/downloads/1.7.2/index.html'>1.7.2</a><br/>
-  <a href='/files/downloads/1.7.3/index.html'>1.7.3</a><br/>
-  <a href='/files/downloads/1.7.4/index.html'>1.7.4</a><br/>
-  <a href='/files/downloads/1.7.5/index.html'>1.7.5</a><br/>
-  <a href='/files/downloads/1.7.6/index.html'>1.7.6</a><br/>
-  <a href='/files/downloads/1.7.7/index.html'>1.7.7</a><br/>
-  <a href='/files/downloads/1.7.8/index.html'>1.7.8</a><br/>
-  <a href='/files/downloads/1.7.9/index.html'>1.7.9</a><br/>
-  <a href='/files/downloads/9.0.0.0.pre1/index.html'>9.0.0.0.pre1</a><br/>
-  <a href='/files/downloads/9.0.0.0.pre2/index.html'>9.0.0.0.pre2</a><br/>
-  <a href='/files/downloads/9.0.0.0.rc1/index.html'>9.0.0.0.rc1</a><br/>
-  <a href='/files/downloads/9.0.0.0.rc2/index.html'>9.0.0.0.rc2</a><br/>
-  <a href='/files/downloads/9.0.0.0/index.html'>9.0.0.0</a><br/>
-  <a href='/files/downloads/9.0.1.0/index.html'>9.0.1.0</a><br/>
-  <a href='/files/downloads/9.0.2.0/index.html'>9.0.2.0</a><br/>
-  <a href='/files/downloads/9.0.3.0/index.html'>9.0.3.0</a><br/>
-  <a href='/files/downloads/9.0.4.0/index.html'>9.0.4.0</a><br/>
-  <a href='/files/downloads/9.0.5.0/index.html'>9.0.5.0</a><br/>
-  <a href='/files/downloads/9.1.0.0/index.html'>9.1.0.0</a><br/>
-  <a href='/files/downloads/9.1.1.0/index.html'>9.1.1.0</a><br/>
-  <a href='/files/downloads/9.1.2.0/index.html'>9.1.2.0</a><br/>
-  <a href='/files/downloads/9.1.3.0/index.html'>9.1.3.0</a><br/>
-  <a href='/files/downloads/9.1.4.0/index.html'>9.1.4.0</a><br/>
-  <a href='/files/downloads/9.1.5.0/index.html'>9.1.5.0</a><br/>
-  <a href='/files/downloads/9.1.6.0/index.html'>9.1.6.0</a><br/>
-  <a href='/files/downloads/9.1.7.0/index.html'>9.1.7.0</a><br/>
-  <a href='/files/downloads/9.1.8.0/index.html'>9.1.8.0</a><br/>
-  <a href='/files/downloads/9.1.9.0/index.html'>9.1.9.0</a><br/>
-  <a href='/files/downloads/9.1.10.0/index.html'>9.1.10.0</a><br/>
-  <a href='/files/downloads/9.1.11.0/index.html'>9.1.11.0</a><br/>
-  <a href='/files/downloads/9.1.12.0/index.html'>9.1.12.0</a><br/>
-  <a href='/files/downloads/9.1.13.0/index.html'>9.1.13.0</a><br/>
-  <a href='/files/downloads/9.1.14.0/index.html'>9.1.14.0</a><br/>
-  <a href='/files/downloads/9.1.15.0/index.html'>9.1.15.0</a><br/>
+
+  <table>
+    <thead>
+      <tr>
+        <th>JRuby version</th>
+        <th>Ruby version</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href='/files/downloads/0.9.0/index.html'>0.9.0</a></td>
+        <td></td>
```

Comment:
The lack of information here and subsequent rows should be addressed somehow—by having a separate table with no compatibility information, by completing the trade arch, or by some other means.